app/assets/javascripts/discourse/app/components/sidebar/more-section-link.hbs

```diff
@@ -1,15 +1,30 @@
-<Sidebar::SectionLink
-  @shouldDisplay={{@sectionLink.shouldDisplay}}
-  @linkName={{@sectionLink.name}}
-  @route={{@sectionLink.route}}
-  @href={{@sectionLink.href}}
-  @query={{@sectionLink.query}}
-  @title={{@sectionLink.title}}
-  @content={{@sectionLink.text}}
-  @currentWhen={{@sectionLink.currentWhen}}
-  @badgeText={{@sectionLink.badgeText}}
-  @model={{@sectionLink.model}}
-  @models={{@sectionLink.models}}
-  @prefixType={{@sectionLink.prefixType}}
-  @prefixValue={{@sectionLink.prefixValue}}
-/>
\ No newline at end of file
+{{#if @sectionLink.shouldDisplay}}
+  {{#if @sectionLink.external}}
+    <Sidebar::SectionLink
+      @linkName={{@sectionLink.name}}
+      @content={{replace-emoji @sectionLink.text}}
+      @prefixType="icon"
+      @prefixValue={{@sectionLink.prefixValue}}
+      @href={{@sectionLink.value}}
+    />
```

Comment:
More section has to accept external links as with this feature those type of links can be added